src/EmitType.re

```diff
@@ -350,18 +350,21 @@ and renderFields =
   let exact =
     config.language == Flow && !config.exportInterfaces && closedFlag == Closed;
   let space = indent == None && fields != [] ? " " : "";
-  ((exact ? "{|" : "{") ++ space)
-  ++ String.concat(
-       config.language == TypeScript ? "; " : ", ",
-       List.map(
+  let renderedFields =
+    fields
+    |> List.map(
          renderField(
            ~config,
            ~indent=indent1,
            ~typeNameIsInterface,
            ~inFunType,
          ),
-         fields,
-       ),
+       );
+  let dotdotdot = config.language == Flow && !exact ? ["..."] : [];
```

Comment:
For the output formatting I'd consider making this `[Indent.break(~indent=indent1) ++ "..."]`

Reply:
Good catch.